packages/ragbits-evaluate/src/ragbits/evaluate/dataset_generator/tasks/answer_gen_task.py

```diff
@@ -0,0 +1,50 @@
+from typing import Any
+
+from distilabel.steps.tasks import TextGeneration
+from ragbits.core.prompt.base import ChatFormat
+
+from ragbits.evaluate.dataset_generator.prompt_basic_answer_gen import BasicAnswerGenPrompt, BasicAnswerGenInput
+
+class AnswerGenTask(TextGeneration):
+    """
+    A task for generating basic answers to questions based on a provided text chunk. This class extends
+    the `TextGeneration` task from the `distilabel` package.
+    """
+
+    @property
+    def inputs(self) -> list[str]:
+        """Defines the input fields required for this task."""
+        return ["chunk", "question"]
+
+    def format_input(self, input: dict[str, Any]) -> ChatFormat:
+        """
+        Formats the input data for the text generation model by creating a `ChatFormat` object.
+
+        Args:
+            input: A dictionary containing "chunk" (text source) and "question" keys.
+
+        Returns:
+            The formatted chat object containing the inputs for text generation.
+        """
+        chat = BasicAnswerGenPrompt(BasicAnswerGenInput(chunk=input["chunk"], question=input["question"])).chat
```

Comment:
i don't think that we should stick to any hardcoded prompt class for a task - prompt class should imo be an instance attribute